prompt/prompt_utils.go

```diff
@@ -51,17 +51,18 @@ func extractReadableTextFromURL(urlString string) (string, error) {
 	return article.TextContent, nil
 }
 
-// TODO promt builder should not be only with a file
 func FilePromptBuilder(prePrompt string, filePaths []string, urls []string, googleDriveFiles []string, userPrompt string) (string, error) {
 	var files []File
 
 	// Read file contents and populate the files slice
 	for _, filePath := range filePaths {
-		fileContent, err := readFileContent(filePath)
-		if err != nil {
-			return "", err
+		if filePath != "" {
```

Comment:
if filepath here is "" it is an error, we should not be hiding it, something is wrong somewhere

Reply:
it is not an error, for example, if you use google documents, in that case, input files could be empty.